src/ResourceManager/Sql/Commands.Sql/Common/AzureSqlCmdletBase.cs

```diff
@@ -59,87 +36,5 @@ protected virtual string GetResourceId(M model)
             HelpMessage = "The name of the resource group.")]
         [ValidateNotNullOrEmpty]
         public string ResourceGroupName { get; set; }
```

Comment:
@akromm @shahabhijeet changing a parameter from mandatory to non-mandatory is not a breaking change; however, the other way around is a breaking change